config/infra-dev/nextflow-aurora-mysql.yaml

```diff
@@ -15,6 +15,7 @@ parameters:
   AccountAdminArns:
     - {{stack_group_config.sso_admin_role.arn}}
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
+    - !stack_output_external github-oidc-nextflow-infra::ProviderRoleArn
```

Comment:
This is referencing the stack in the strides account, but everything under `infra-dev/` should reference the stack in the dev account: `sagebase-github-oidc-workflows-dev-nextflow-infra::ProviderRoleArn`

Reply:
fixed